examples/simple/test/fixtures/bwb_spec.json

```diff
@@ -24,6 +24,7 @@
     },
     "label": "//test/fixtures:xcodeproj_bwb.generator",
     "name": "bwb",
+    "pre_build_script": null,
```

Comment:
Doesn't seem to be working?

Reply:
I'm not sure why running `test/update_all_fixtures.sh` on my machine doesn't update this, but produces a lot of untracked files. I'm seeing the pre-build phase generated correctly in the project by running:

```
cd examples/simple
bazel run xcodeproj
```

Reply:
This file has the correct contents:

```
examples/simple/bazel-output-base/execroot/__main__/bazel-out/darwin-dbg/bin/xcodeproj.generator_spec.json
```

```
{"bazel_config":"rules_xcodeproj","bazel_workspace_name":"__main__","build_settings":{"ALWAYS_SEARCH_USER_PATHS":false,"BAZEL_PATH":"bazel","CLANG_ENABLE_OBJC_ARC":true,"CLANG_MODULES_AUTOLINK":false,"COPY_PHASE_STRIP":false,"ONLY_ACTIVE_ARCH":true,"USE_HEADERMAP":false,"VALIDATE_WORKSPACE":false},"configuration":"darwin-dbg","custom_xcode_schemes":[],"extra_files":["BUILD"],"force_bazel_dependencies":false,"index_import":{"t": "g", "_": "darwin-opt-exec-2B5CBBC6/bin/external/rules_xcodeproj_index_import/index-import"},"label":"//:xcodeproj.generator","name":"Simple","pre_build_script":"pre build.sh","replacement_labels":[],"scheme_autogeneration_mode":"auto","target_hosts":[],"target_merges":[],"targets":["//:SwiftBin darwin-dbg-ST-5d7f514c2b25",{"build_settings":{"DEBUG_INFORMATION_FORMAT":"dwarf","ENABLE_BITCODE":false,"ENABLE_TESTABILITY":true,"GCC_OPTIMIZATION_LEVEL":"0","OTHER_SWIFT_FLAGS":"-Xcc -O0 -Xcc -DDEBUG=1 -Xcc -fstack-protector -Xcc -fstack-protector-all","PRODUCT_MODULE_NAME":"SwiftBinModuleName","SWIFT_ACTIVE_COMPILATION_CONDITIONS":"DEBUG HI","SWIFT_OBJC_INTERFACE_HEADER_NAME":"","SWIFT_OPTIMIZATION_LEVEL":"-Onone","SWIFT_VERSION":"5"},"configuration":"darwin-dbg-ST-5d7f514c2b25","inputs":{"srcs":["main.swift"]},"label":"//:SwiftBin","linker_inputs":{"linkopts":["-headerpad_max_install_names","-no-canonical-prefixes","-lc++"]},"lldb_context":{"c":[{"o":["-O0","-DDEBUG=1","-fstack-protector","-fstack-protector-all"],"q":[".",{"_":"darwin-dbg-ST-5d7f514c2b25/bin","t":"g"}]}]},"name":"SwiftBin","outputs":{"p":{"_":"darwin-dbg-ST-5d7f514c2b25/bin/rules_xcodeproj/SwiftBin/SwiftBin","t":"g"}},"package_bin_dir":"bazel-out/darwin-dbg-ST-5d7f514c2b25/bin","platform":{"arch":"x86_64","minimum_os_version":"12.3","os":"macos","variant":"macosx"},"product":{"executable_name":"SwiftBin","name":"SwiftBin","path":{"_":"darwin-dbg-ST-5d7f514c2b25/bin/rules_xcodeproj/SwiftBin/SwiftBin","t":"g"},"type":"com.apple.product-type.tool"},"search_paths":{"quote_includes":[".",{"_":"darwin-dbg-ST-5d7f514c2b25/bin","t":"g"}]}}]}
```

Reply:
I'll take a look again tomorrow.

Reply:
You need to update the fixture definition. Set the value in` xcode_targets.bzl` and reuse that in both places.